src/Command/PopulateCommand.php

```diff
@@ -171,13 +170,6 @@ private function populateIndex(OutputInterface $output, string $index, bool $res
             }
         );
 
-        $this->dispatcher->addListener(
```

Comment:
I think this change needs to be reverted, otherwise custom event listeners won't be triggered.

Reply:
I'm not sure what you mean here. This command registers its own listener for an event (that is never emitted at all). This listener registration does not create any event by itself?